coderedcms/models/page_models.py

```diff
@@ -1291,24 +1227,22 @@ def process_form_submission(self, request, form):
         for fn in hooks.get_hooks('form_page_submit'):
             fn(instance=self, form_submission=form_submission)
 
-        return processed_data
-
     def send_summary_mail(self, request, form, processed_data):
         """
         Sends a form submission summary email.
         """
         addresses = [x.strip() for x in self.to_address.split(',')]
         content = []
-        for field in form:
-            value = processed_data[field.name]
-            # Convert lists into human readable comma separated strings.
+
+        for key, value in processed_data.items():
             if isinstance(value, list):
                 value = ', '.join(value)
             content.append('{0}: {1}'.format(
-                field.label,
+                key.replace('-', ' ').replace('_', ' ').title(),
```

Comment:
It's not very DRY-ish that we are doing the string replace here, and also in the `data_to_dict` method. Is there a way to improve?